reconcile/terraform_vpc_peerings.py

```diff
@@ -60,11 +67,11 @@ def _get_default_management_account(
 
 
 def _build_infrastructure_assume_role(
-    account: dict[str, Any],
-    cluster: dict[str, Any],
-    ocm: Optional[OCM],
-    provided_assume_role: Optional[str],
-) -> Optional[dict[str, Any]]:
+    account: Mapping[str, Any],
+    cluster: Mapping[str, Any],
+    ocm: OCM | None,
+    provided_assume_role: str | None,
+) -> Mapping[str, Any] | None:
```

Comment:
return type can be concrete type `dict[str, Any] | None`, generic in, concrete out.

Reply:
Fixed in [daa3332](https://github.com/app-sre/qontract-reconcile/pull/4381/commits/daa3332b0ef1a46304faafb082ce85dd382be201)